CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.6.5
```

Comment:
Ok..I bumped the gem version number in https://github.com/twitter/secureheaders/pull/334/commits/d2070ea581004db8b0b3a83550ffccccb46c8846.